include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -1709,8 +1709,16 @@ class FEEvaluationAccess<1, 1, Number, is_face, VectorizedArrayType>
  * setting the macro `FE_EVAL_FACTORY_DEGREE_MAX` to the desired integer and
  * instantiating the classes FEEvaluationFactory and FEFaceEvaluationFactory
  * (the latter for FEFaceEvaluation) creates paths to templated functions for
- * a possibly larger set of degrees. You can check if fast
- * evaluation/integration for a given degree/n_quadrature_points pair by calling
+ * a possibly larger set of degrees. This can both be set when configuring
+ * deal.II by passing the flag `-D FE_EVAL_FACTORY_DEGREE_MAX=8` (in case you
+ * want to compile all degrees up to eight; recommended setting) or by
+ * compiling `evaluation_template_factory.templates.h` and
+ * `evaluation_template_face_factory.templates.h` with the
+ * `FE_EVAL_FACTORY_DEGREE_MAX` overriden to the desired value. In the second
+ * option, symbols will be available twice, and it depends on your linker and
+ * dynamic library loader whether the user-specified setting takes
+ * precendence. You can check if fast evaluation/integration for a given
```

Comment:
I think this is still pretty brittle and I'm not sure we really want to encourage people to do that. Maybe, mention `LD_PRELOAD` at least?

Reply:
Yes, `LD_PRELOAD` is a good idea.

Reply:
I can also remove the second recommendation altogether, if we prefer that. I recommend the other option, which I wanted to achieve.